lib/modules/dosomething/dosomething_mbp/dosomething_mbp.install

```diff
@@ -144,6 +144,14 @@ function dosomething_mbp_update_7008() {
 }
 
 /**
+ * Updates mbp productions to include messagingGroupsQueue.
+ */
+function dosomething_mbp_update_7009() {
+  db_drop_table('message_broker_producer_productions');
```

Comment:
What does this table store? Want to double check this won't affect the resurrection of mailchimp groups that @justkika was looking to implement for Truth